scripts/macos.sh

```
@@ -497,6 +497,14 @@ defaults write com.google.Chrome.canary PMPrintingExpandedStateForPrint2 -bool t
 sudo scutil --set ComputerName "Shun-Tagami-MacBook"
 sudo scutil --set LocalHostName "Shun-Tagami-MacBook"
 
+###############################################################################
+# iTerm2                                                                      #
+###############################################################################
+
+# Load preferences from custom folder
+defaults write com.googlecode.iterm2 PrefsCustomFolder -string "~/dotfiles/misc"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)